src/main/java/ru/hogwarts/school/service/AvatarService.java

```diff
@@ -0,0 +1,12 @@
+package ru.hogwarts.school.service;
+
+import org.springframework.web.multipart.MultipartFile;
+import ru.hogwarts.school.model.Avatar;
+
+import java.io.IOException;
+
+public interface AvatarService {
+    void uploadAvatar(Long id, MultipartFile file) throws IOException;
```

Comment:
```suggestion
    Long uploadAvatar(Long id, MultipartFile file) throws IOException;
```
Тут тоже поменяй на Long